jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdeMsgPayload.java

```diff
@@ -1,89 +1,67 @@
 /*******************************************************************************
  * Copyright 2018 572682
- * 
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not
  * use this file except in compliance with the License.  You may obtain a copy
  * of the License at
- * 
  *   http://www.apache.org/licenses/LICENSE-2.0
- * 
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
  * License for the specific language governing permissions and limitations under
  * the License.
  ******************************************************************************/
+
 package us.dot.its.jpo.ode.model;
 
-public class OdeMsgPayload extends OdeObject {
+import lombok.Data;
+import us.dot.its.jpo.ode.util.JsonUtils;
+
+/**
+ * Generic payload object for ODE messages.
+ *
+ * @param <T> the type of data contained in the payload
+ */
+@Data
+public class OdeMsgPayload<T> {
```

Comment:
Made the base class type paramaterized to allow for `jpo-asn-pojos` classes to be used in the payload.